### DISCUSSION TITLE
Q: 请问有没有方法给过长的评论添加展开功能？ || Q: Is there any way to add an expansion function to comments that are too long?

### DISCUSSION THREAD

**Discussion by Tseshongfeeshur** [walinejs/waline#2088] [Q&A]
### Describe the bug | 问题描述

由于网站特殊性，可能会出现太长的评论，影响观感。请问有没有方法给过长的评论添加展开功能？也就是说可以在较短的篇幅预览，点击展开时即可浏览全文。

### Website URL | 问题网站

https://596forever.github.io/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Due to the particularity of the website, comments that are too long may appear, affecting the viewing experience. Is there any way to add an expansion function to comments that are too long? That is to say, you can preview it in a shorter length and click to expand it to browse the full text.

### Website URL | Problem website

https://596forever.github.io/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
感觉不是特别好实现

---
It doesn’t feel particularly easy to implement
